src/basic/time-util.c

```diff
@@ -381,19 +399,11 @@ static char *format_timestamp_internal(
 }
 
 char *format_timestamp(char *buf, size_t l, usec_t t) {
```

Comment:
maybe move this function to the header, make it static inline and just a wrapper around format_timestamp_style() with TIMESTAMP_PRETTY

Reply:
done